src/utils.cc

```diff
@@ -184,3 +188,35 @@ std::vector<std::string> utils::SpiltWithSymbol(const std::string& str,
   free(input);
   return str_list;
 }
+std::string utils::GetCommandExecuteResult(const std::string& command) {
+  std::string result;
+  CommandExecutor commandExecutor;
+  int save_fd_out = dup(STDOUT_FILENO);
+  int save_fd_err = dup(STDERR_FILENO);
+  std::string temporary_file = "file.txt";
+  int fd_out =
+      open(temporary_file.c_str(), O_WRONLY | O_APPEND | O_CREAT, 0664);
+  dup2(fd_out, STDOUT_FILENO);
+  dup2(fd_out, STDERR_FILENO);
```

Comment:
Can we just add output direction to command? Like `command.append("> " + temporary_file);` Since our `Execute` can handle output direction already, so we may not need to do low level redirection here.

Reply:
When using a file redirected by'>', it will cause the error to be printed on the screen, but standard bash does not redirect the error to the file